pyanaconda/modules/common/structures/group.py

```diff
@@ -65,5 +64,14 @@ def gid(self) -> Int:
         return self._gid
 
     @gid.setter
-    def gid(self, gid: Int):
+    def gid(self, gid: UInt32):
         self._gid = gid
+
+    @property
+    def gid_status(self) -> Str:
```

Comment:
Please, define these properties before the original ones since these properties decide whether the original ones will be used or not. It is just easier to read the code that way.

Reply:
Done.